lib/content-services/src/lib/breadcrumb/breadcrumb.component.html

```diff
@@ -57,7 +57,7 @@
         </a>
 
         <div *ngSwitchDefault class="adf-breadcrumb-item-current" aria-current="location">
-            {{ item.name | translate }}
+            {{selectedRowItemsCount < 1 ? (item.name | translate) : ('BREADCRUMB.HEADER.SELECTED' | translate: { count: selectedRowItemsCount }) }}
```

Comment:
could you please add space before selectedRowItemsCount? Like: 
{{ selectedRowItemsCount

Reply:
@jacekpluta you changed it to number but link still indicates data column model